ejb/ejb2-connector/ejb2-connector-1.0.0/ejb-connector/src/main/assembly/assemble-connector.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Folder structure is wrong. It should be like 'ejb2/ejb2-connector/ejb2-connector-1.0.0/org.wso2.carbon.connector'.